service/templates/layout.html

```diff
@@ -41,7 +41,7 @@
     <div class="phase-banner">
       <p>
         <strong class="phase-tag">BETA</strong>
-        <span>This is a new service - your <a href="mailto:digital-register-feedback@digital.landregistry.gov.uk">feedback</a> will help us to improve it.</span>
+        <span>This service is in private beta. Please give us your <a rel="external" href="mailto:digital-register-feedback@digital.landregistry.gov.uk">Feedback</a>.</span>
```

Comment:
@AndyPorter as well as adding back in the word "private", the wording here has been changed to an earlier version. Is this correct? Proto 18 went with words along the lines of "your feedback will help us to improve it"